### ISSUE TITLE
Sometimes the build-time linting takes too much time

### ISSUE THREAD

**Issue by vlaaad** [defold/defold#8989]
**Describe the bug (REQUIRED)**
Sometimes, the development iteration process is slowed down by the project code linting. We should fix it.

**To Reproduce (REQUIRED)**
Would be nice to have these...

**Expected behavior (REQUIRED)**
Linting does not take too much time.

**Workaround (OPTIONAL):**
We could implement disabling the workspace linter, see #8974

**Comment by jalhund** [comment]
Yes, linting definitely should be switchable. It's too annoying when you have to wait even if only one line of code was changed

**Comment by Anisin** [comment]
We have a fairly large project, about 2400 lua files.
When we moved to 1.9.0 and I started the Linting Code stage (Ctrl+B) it took about 20-60 seconds to start the build.
When I first launched the project, the total time to Linting took 40-50 sec.
In other conditions Linting takes 10-20 sec. but sometimes it goes up to +/- 40 sec. again.
If I disable Linting Code functionality, the build starts instantly.

**Comment by AGulev** [comment]
I turn it off by default for now https://github.com/defold/defold/pull/13283
